tools/pfe-tools/dev-server/plugins/pfe-dev-server.ts

```diff
@@ -97,24 +97,20 @@ function getRouter(options: PfeDevServerInternalConfig) {
   const { elementsDir, tagPrefix } = options;
   const { componentSubpath } = options.site;
   const router = new Router()
+
+      // PFE TOOLING
       .get('/tools/pfe-tools/environment.js(.js)?', async ctx => {
         ctx.body = await makeDemoEnv(options.rootDir);
         ctx.type = 'application/javascript';
       })
 
-  // Redirect `components/jazz-hands/*-lightdom.css` to `elements/pf-jazz-hands/*-lightdom.css`
-  // NOTE: don't put subresources in /demo called `*-lightdom.css` , or this will break
-      .get(`/${componentSubpath}/:element/(demo/)?:fileName.css`, async (ctx, next) => {
-        const { element, fileName } = ctx.params;
-        if (!element.startsWith(tagPrefix) && fileName.includes('lightdom')) {
-          const prefixedElement = deslugify(element);
-          ctx.redirect(`/${elementsDir}/${prefixedElement}/${fileName}.css`);
-        } else {
-          return next();
-        }
```

Comment:
I found this block to be unnecessary and a much more simple check in the `components/:element/*`  => `elements/:element/*` block later down in the code